dGame/dMission/Mission.cpp

```diff
@@ -482,7 +482,8 @@ void Mission::YieldRewards() {
     }
 
     if (info->reward_currency > 0) {
-        character->SetCoins(character->GetCoins() + info->reward_currency, info->isMission);
+        int32_t lootSource = info->isMission ? LOOT_SOURCE_MISSION : LOOT_SOURCE_ACHIEVEMENT;
```

Comment:
```suggestion
        eLootSourceType lootSource = info->isMission ? LOOT_SOURCE_MISSION : LOOT_SOURCE_ACHIEVEMENT;
```
Missed one, sorry. Fortunately, there was a warning in the automatic checks. Unfortunately, I don't think you can see that 🤔

Reply:
Noticed this one while I was compiling the changes.  Already pushed the change